main.jsx

```diff
@@ -24,36 +24,11 @@ import Navbar from "./components/navbar/navbar.jsx";
 import Footer from "./components/footer/footer.jsx";
 
 const Featured = () => {
```

Comment:
I kept the Featured component & `<Route>` in case we wanted to utilize it in the future. 🤷‍♀

Reply:
We are unlikely going to _toggle_ the Featured component in the future so let's remove its related code if it's not in use.